calaccess_raw/management/commands/__init__.py

```diff
@@ -20,6 +45,109 @@ def handle(self, *args, **options):
         self.verbosity = options.get("verbosity")
         self.no_color = options.get("no_color")
 
+        self.start_datetime = datetime.now()
+
+        # TODO: see if there's another way to identify caller
+        # in (edge) case when update is not called from command line
+        # if called from command line
+        if self._called_from_command_line:
+            # use the arg passed to manage.py as command name
+            self.command_name = sys.argv[1]
+        else:
+            # or take the end of the command's full module name
+            self.command_name = sub(r'(.+\.)*', '', self.__class__.__module__)
+
+        self.database = options["database"] or 'default'
+
+        self.raw_data_versions = RawDataVersion.objects.using(self.database)
+        self.raw_data_files = RawDataFile.objects.using(self.database)
+        self.command_logs = CalAccessCommandLog.objects.using(self.database)
```

Comment:
Ah, right, I didn't see the assignment there. Yes, that `else` statement looks good!

>I'm pretty sure that, if the user wants to log records in db other than default, we have to manually set the db.

Yep!

>The suggestion coming from db_for_write() router method for any of the tracking models will be 'default'.

That's true for the default router, but isn't true if you implement your own and point Django to your router in `settings.py`. E.g.

https://github.com/caciviclab/disclosure-backend/blob/master/disclosure/routers.py
https://github.com/caciviclab/disclosure-backend/blob/master/disclosure/settings.py#L96

With the router, I can pick the default database for any model. I do that to shove all `CalAccess` models into a `calaccess_raw` database, for example, to keep our table schema in our default database simpler.